Public/Sdk/Public/Deployment/filesystem.dsc

```diff
@@ -16,6 +16,29 @@ export interface OnDiskDeployment {
 
     /** Optional primary file, i.e. an executable or test dll */
     primaryFile?: File;
+
+    /** Optional opaque directories robocopied/rsynced into this deployment */
+    targetOpaques?: OpaqueDirectory[];
+}
+
+@@public
+export interface OpaqueSubDirectory {
+    /** Parent opaque directory */
+    opaque: OpaqueDirectory,
+
+    /** A path relative to that opaque directory designating a subdirectory to be added to the deployment */

```

Comment:
>A path relative to that op [](start = 8, length = 26)

I'd say 'An optional path to...' and 'If not specified, the whole opaque directory will be added to the deployment'